### DISCUSSION TITLE
issue: ukrainian i18n font

### DISCUSSION THREAD

**Discussion by OriginalSimon** [open-webui/open-webui#1556] [General]
**Bug Summary:**
In the Ukrainian localization, font inconsistency is apparent across various sections of the application. While some text segments maintain a smooth and aesthetically pleasing appearance, others, including registration and authorization windows, as well as the administrator panel, exhibit a sharp font resembling "Times New Roman." This inconsistency disrupts the overall design impression and needs to be addressed for uniformity throughout the application.

**Additional Information:**
The observed font irregularity seems to stem from the rendering of Cyrillic characters. It's crucial to ensure that the fonts being used adequately support Cyrillic characters to maintain design cohesion and readability.

Interesting observation: on my phone, all the fonts are the same. I also checked this error on PC in two browsers Firefox and Microsoft Edge, the error was in both.

**Comment by tjbck** [comment]
@SimonOriginal could you share a screenshot of where exactly the font looks broken?

**Comment by OriginalSimon** [comment]
I see that the interface has been updated in the above items and the problem has been solved it, only the registration and authorisation window remains. Screenshot attached.

![Image 1](https://github.com/open-webui/open-webui/assets/94782611/bf650465-9d4b-488a-b5bf-67692d5e0b75)
![Image 2](https://github.com/open-webui/open-webui/assets/94782611/a97c93f0-0247-48dc-858a-4b90eeeb8c1e)

**Reply by tjbck** [reply]
Will be fixed in our next release!
